File: VB.NET/Aula02_atividade02/Aula02_atividade02/Professor.vb
Public Class Professor
    Inherits Pessoa
    Private titulo As String

    Public Sub New(nome As String, titulo As String)
        MyBase.New(nome)
        Me.titulo = titulo
    End Sub

    Public Property pTitulo As String
        Get
            Return titulo
        End Get
        Set(value As String)
            titulo = value
        End Set
    End Property
End Class

File: C#/Aula01/Aula01/Professor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Aula01
{
    class Professor:Pessoa
    {
        private String titulo;

        public String Titulo
        {
            get { return titulo; }
            set { titulo = value; }
        }

        public Professor(String cpf, String nome, String titulo)
        {
            this.CPF = cpf;
            this.Nome = nome;
            this.Titulo = titulo;
        }
    }
}
